src/ImportExport.ts

```diff
@@ -173,6 +224,12 @@ export const exportSynergism = async () => {
         : 'Savefile copied to file!';
 }
 
+export const errorGame = async () => {
+    await Alert('sorry. You may be using an older version of your browser, current save data may be incorrect, or some problem may prevent from starting the game\nThis button just deletes the save data, but Synergism will work\nThen import the exported save data have\nCan report the console in error on Discord Synergism channel');
```

Comment:
```suggestion
    
```
This message makes it seem as though there was an issue deleting the save. Probably better to remove it as the alert below is good enough.